sample/cppref-sample.cpp

```diff
@@ -10,17 +10,24 @@
  * license.  You may distribute this file (and only this file) under
  * that license, see http://creativecommons.org/licenses/by-sa/3.0/.
  *
- * Also licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with a valid License.
- * You may obtain a copy of the Apache License at
+ * Also licensed under the MIT License:
```

Comment:
Wasn't completely sure on the dual-licensing of this file, this was the only place that the exact same patch did not apply cleanly.